desktop/renderer-app/typings/global.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="vite/client" />
```

Comment:
Shall we use import instead of reference here?

FYI: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/triple-slash-reference.md

Reply:
better use `types: ["vite/client"]` in tsconfig